projects.go

```diff
@@ -282,6 +282,35 @@ func (s *ProjectsService) ListProjectsUsers(pid interface{}, opt *ListProjectUse
 	return p, resp, err
 }
 
+// ProjectLanguages is a map of strings because the response is arbitrary
+// Gitlab API docs with example: https://docs.gitlab.com/ce/api/projects.html#languages
+type ProjectLanguages map[string]interface{}
```

Comment:
Reading the docs it seems to me this should be a `map[string]float`

Reply:
You are right! thanks!